### ISSUE TITLE
What does `time make -f freshen.mak` do?

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/quickBlocks#507]
At this line (https://github.com/Great-Hill-Corporation/quickBlocks/blob/develop/src/monitors/scripts/watchContract.py#L33) of watchContract.py there is a question as to what the `time make -f freshen.mak` command does.

At the top level of the quickBlocks folder is this folder: https://github.com/Great-Hill-Corporation/quickBlocks/tree/master/monitors.  Computers with quickBlocks monitor code installed will have a file in this folder (and a series of subfolder) that look something like this:

    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:18 augur
    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:19 bnt
    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:19 dao
    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:19 ENS
    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:19 firstBlood
    drwxr-xr-x  6 jrush  staff  204 Sep 12 23:19 gnosis
    drwxr-xr-x  6 jrush  staff  204 Sep 27 22:24 singular
    -rw-r--r--@ 1 jrush  staff  546 Oct  6 22:51 freshen.mak

Where each folder contains the monitor code for a given smart contract, and freshen.mak (at this level) looks like this:

    freshen:
        @echo ; cd augur ; bin/augur freshen -v
        @echo ; cd bnt ; bin/bnt freshen -v --trace
        @echo ; cd dao ; bin/dao freshen -v
        @echo ; cd ENS ; bin/ENS freshen -v
        @echo ; cd firstBlood ; bin/firstBlood freshen -v
        @echo ; cd gnosis ; bin/gnosis freshen -v
        @echo ; cd singular ; bin/singular freshen -v

So it just changes into each monitor folder and runs the monitor (it does this every 20 seconds or so--whatever the watchContracts code tells it to do.

The monitor code itself (./augur/bin/augur, ./gnosis/bin/gnosis) is run from the current folder an writes data after analyzing the recent blocks (that were accumulated with the blockScrape, blockAcct, and miniBlock programs). [Ignore that this is inefficient--this is how it currently works, but this will change over time as I figure things out.]

In summary, I use the makefile to run each monitor at each spin of the wheel of watchContacts. The outputs of these programs should go to the screen.

**Comment by tjayrush** [comment]
You can fix this for one folder, and I will fix it for the rest. To fix it, the line should change into the ./monitors/augur folder (so the next step has a pwd of ./monitors/augur) and then run (from there) ./bin/augur and output its output to the screen.

**Comment by crodnun** [comment]
Ok clear now, should be implemented and pushed to repository the changes required to accomodate this logic. Kindly review the script and feel to close this issue and continue in this one #505

**Comment by tjayrush** [comment]
Closing
